packages/astro-rss/src/util.ts

```diff
@@ -0,0 +1,19 @@
+import npath from 'path-browserify';
+
+/** Normalize URL to its canonical form */
+export function createCanonicalURL(url: string, base?: string): URL {
+	let pathname = url.replace(/\/index.html$/, ''); // index.html is not canonical
+	pathname = pathname.replace(/\/1\/?$/, ''); // neither is a trailing /1/ (impl. detail of collections)
```

Comment:
Why drop trailing `/1/`? That breaks my url : "mysite.com/episodes/1/", thus redirecting people to the episodes collection instead of the actual first episode.

I use collections, and with a file named `1.md`, it works just fine in the site itself. It's just that this RSS plugin won't reproduce the URL properly.

Reply:
...that's an excellent question 😅 I'm not sure! This seems like legacy could that definitely shouldn't be here.

Reply:
PR to remove: https://github.com/withastro/astro/pull/6970